dependencies.gradle

```diff
@@ -70,6 +70,7 @@ ext.versions = [
         tensorflowLite              : '2.11.0',
         tensorflowLiteSupport       : '0.4.3',
         testParameterInjector       : '1.17',
+        timber                      : '5.0.1',
```

Comment:
We have a [Logger](https://github.com/stripe/stripe-android/blob/f837bac7/stripe-core/src/main/java/com/stripe/android/core/Logger.kt#L7) class in stripe-core. Would this fit your needs?

Reply:
Yes, this looks great! My goal is to demonstrate reasonable logging behavior in our example app - I've updated the PR to remove Timber.